etc/kapacitor/kapacitor.conf

```diff
@@ -275,6 +275,14 @@ default-retention-policy = ""
   # The URL for the Pushover API.
   url = "https://api.pushover.net/1/messages.json"
 
+##########################################
+# Configure Alert POST request Endpoints
+
+# [[alertpost]]
+#   endpoint = "example"
+#   url = "http://example.com"
+#   headers = { Authorization = "your-key" }
+
```

Comment:
Good question, have we tried it? I think there is support for maps in our env code but we should test it. Even add a test to the server package for it.